cli/command/manifest/rm.go

```diff
@@ -0,0 +1,47 @@
+package manifest
+
+import (
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type rmOpts struct {
+	target string
+}
+
+func newRmManifestListCommand(dockerCli command.Cli) *cobra.Command {
+	opts := rmOpts{}
+
+	cmd := &cobra.Command{
+		Use:   "rm MANIFEST_LIST",
+		Short: "Delete a manifest list from local storage",
+		Args:  cli.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.target = args[0]
```

Comment:
Should it accept multiple manifests? (i.e., `docker manifest rm one two three`) other `rm` commands support multiple manifests so it would probably make sense to accept that for this command as well